chat.py

```diff
@@ -35,27 +35,38 @@ def Send(name, sock:socket, host, port2, messages, q):
 
 def Recieve(sock:socket, host, port, messages, q):
 	templist = list()
+	confirm = ''
+	first = False
+	second = False
 	while True:
 		try:
 			data,addr = sock.recvfrom(1024)
 
 			if data:
 				#temp = data.decode('utf-8')
 				message = data.decode('utf-8')
+				#print(message)
 				if message[0] == '/':
 					print("Some messages were lost")
 					message = message.replace('/', '')
 					lost_message = messages[message]
 					sock.sendto(lost_message[1].encode('utf-8'), (host, port))
 
-				check_messages(messages, sock, host, port, int(templist[1]))
+				elif message[0] == '#':
+					print("Your message was succesfully recieved")
+					templist = message.split('/')
+					confirm = messages[int(templist[1])]
+					confirm = confirm + ('r', )
+					messages[int(templist[1])] = confirm
```

Comment:
And where do you use this status later?

Reply:
I don't used because I think you just nedded confirmation but I added it because it can be another way for finding messages that were not delivered

Reply:
Why on earth would I want it? Your project needs a way to ensure that all messages were received by the other client.